packages/forms-web-app/src/locales/cy/common.json

```diff
@@ -1,3 +1,7 @@
 {
-	"search": "Chwilio"
+	"search": "Chwilio",
+	"hideButton": "Cuddio",
+	"showButton": "Dangos",
+	"hideAllButton": "Cuddio pob adran",
+	"showAllButton": "Dangos pob adran"
 }
```

Comment:
Would these keys be easier to understand if they matched the translated text?

"hideButton" > "hide", "hideAllButton" > "hideAll" etc.